### ISSUE TITLE
NA west inventory refresh off

### ISSUE THREAD

**Issue by Grazzak** [Xeio/WanderLost#220]
Hello,

Could the inventory reset be adjusted for each region?

NA west resfresh is 5hrs into the merchant spawn 
![image](https://github.com/Xeio/WanderLost/assets/143309755/bd916ddc-d9f7-4c61-8c87-384a63d19779)

**Comment by Xeio** [comment]
That is... interesting. I wonder if it's the same time for all regions, unrelated to merchant spawn. 🤔

**Comment by Xeio** [comment]
So when you say 5 hours in, that means it's reset only 30 minutes before the merchant despawns?

**Comment by Grazzak** [comment]
It seems like the merchants restock at the same time across all servers but because the spawn times are all different each one ends up with a different restock time.  Here are some comments from other people. It seems NAW& EUC have the same restock time but EUW & NAW have different restocks due to the timezone difference. 
![image](https://github.com/Xeio/WanderLost/assets/143309755/963a73d9-e33e-4666-ab0b-0bb8fdd44015)

**Comment by Grazzak** [comment]
I took a peek into your code as I was curious what you added and saw you had a comment about figuring out the logic for the restock. As it stands right now the the wandering merchant responds at 3am in NAW which seems to coincide with 10am UTC reset and every merchant is set to restock at every 6 hours afterwards. 

Considering all islands, and dailies seem to reset based off it, I would go off it and see if it shifts at the next daylight savings. If you can code it to the 4 6hr resets of 10AM/4PM/10PM/4AM UTC and have the restock timer show the time till that in your local time it should solve an issues in the future.

**Comment by Grazzak** [comment]
I was curious if the 10am UTC reset time would result in a similar restock time as the reddit comments I linked above but it seems to not show the reset anywhere but NAW now.  So I am reopening as I feel I broke the reset for other regions now. 

10am utc results in a 3am pst which coincides with our merchant restock. With some merchants like rowen able to double spawn you cannot buy anything the new vendor provides if you bought it from the previous merchant before reset. It may need testing as I haven't converted it to the other timezones to see if they fit.
